pages/events/_id/index.vue

```diff
@@ -343,13 +343,8 @@ export default {
     }
   },
   setup() {
-    const {
-      uid,
-      can,
-      account,
-      updateAccount,
-      sendSignInLinkToEmail,
-    } = useAuth()
+    const { uid, can, account, updateAccount, sendSignInLinkToEmail } =
```

Comment:
`const { uid, can, account, updateAccount, sendSignInLinkToEmail } = useAuth()`
I think this has to be one line.